test/runtests.jl

```diff
@@ -47,6 +47,10 @@ if Enzyme.EnzymeRules.issupported()
     include("rrules.jl")
     include("kwrules.jl")
     include("kwrrules.jl")
+    if VERSION ≥ v"1.9-"
+        # XXX invalidation does not work on Julia 1.8
```

Comment:
Is this check still needed?

Reply:
Yes. The backedges are added properly on 1.7/1.8, but the invalidation doesn't work as expected. Probably because of some (already fixed) bugs within the Julia runtime system.